Policy support in API (#141)
hashicorp/terraform-docs-common: website/docs/cloud-docs/api-docs/policy-set-params.mdx

# Policy Set Parameters API

> -> **Note:** Sentinel and OPA policies are paid features, available as part of the **Team & Governance** upgrade package. [Learn more about Terraform Cloud pricing here](https://www.hashicorp.com/products/terraform/pricing).

Reviewer: Also would it be possible to add a 1-3 sentence description of what Policy Set parameters mean/are? I'm having trouble understanding exactly what this is and how it relates to all the other APIs. For example, don't we already have one for policy sets? Maybe we should link to that API and explain what's different about this one.

```suggestion
-> **Note:** Sentinel and OPA policies available in the [Terraform Cloud **Team & Governance** tier](https://www.hashicorp.com/products/terraform/pricing), and OPA policies are in beta.
```

# Policy Set Parameters API

> -> **Note:** Sentinel and OPA policies available in the [Terraform Cloud **Team & Governance** tier](https://www.hashicorp.com/products/terraform/pricing), and OPA policies are in beta.

Reviewer: ```suggestion
-> **Note:** Sentinel and OPA policies available in the [Terraform Cloud **Team & Governance** tier](https://www.hashicorp.com/products/terraform/pricing), and OPA policies are in beta. OPA policies are not available in Terraform Enterprise.
```

> Policy Set Parameters allow the ability for policies to receive parameters for use within the policy.
> 
> This set of APIs covers create, update, list and delete operations on parameters.

Reviewer: And then you know what would also be helpful? Just like one sentence of example. like "For example, you could create a parameter to do z, y, z."  Sometimes I think stuff like this can help users visualize why they would want to use this API/why it's helpful.

```suggestion
Policy Set Parameters let you inject parameters into an existing policy. This set of APIs covers create, update, list and delete operations on parameters.
```

# Policy Set Parameters API

> -> **Note:** Sentinel and OPA policies available in the [Terraform Cloud **Team & Governance** tier](https://www.hashicorp.com/products/terraform/pricing), and OPA policies are in beta. OPA policies are not available in Terraform Enterprise.

Reviewer: ```suggestion
-> **Note:** Sentinel and OPA policies available in the [Terraform Cloud Team & Governance tier](https://www.hashicorp.com/products/terraform/pricing), and OPA policies are in beta. OPA policies are not available in Terraform Enterprise.
```

> Policy Set Parameters let you inject parameters into an existing policy. This set of APIs covers create, update, list and delete operations on parameters.
> 
> Sentinel allows a policy authors to supply parameters to help facilitate policy reuse and ensure sensitive values do not need to be hard-coded in a policy.

Reviewer: ```suggestion
[Sentinel parameters](https://docs.hashicorp.com/sentinel/language/parameters) are a list of key/value pairs that Terraform Cloud sends to the Sentinel runtime when performing policy checks on workspaces. They can help you avoid hardcoding sensitive parameters into a policy. 

Parameters are only available for Sentinel policies. This set of APIs provides endpoints to create, update, list and delete parameters.
```
